src/java/org/apache/cassandra/index/sai/disk/format/Version.java

```diff
@@ -54,17 +55,19 @@ public class Version
     public static final Version DB = new Version("db", V4OnDiskFormat.instance, (c, i, g) -> stargazerFileNameFormat(c, i, g,"db"));
     // revamps vector postings lists to cause fewer reads from disk
     public static final Version DC = new Version("dc", V5OnDiskFormat.instance, (c, i, g) -> stargazerFileNameFormat(c, i, g, "dc"));
+    // histograms in index metadata
+    public static final Version EA = new Version("ea", V6OnDiskFormat.instance, (c, i, g) -> stargazerFileNameFormat(c, i, g, "ea"));
```

Comment:
Let's use EB, we want to leave the A version for Apache

Reply:
I think @JeremiahDJordan suggested somewhere that we pick an entirely different scheme to prevent collisions. What if we added a number to the name?

Reply:
Yes. If we want to avoid clashing with Apache we should make our naming convention just different. Like bti vs big that we have for sstables. And then each format has its own version separate from that.
If we don’t want to figure that out right now, I would suggest starting higher up the alphabet than b. Very possible Apache does a minor rev and wants to use b. Start at m or something.